src/layout.js

```diff
@@ -157,6 +157,10 @@ export default {
 	},
 
 	draw: function(chart, labels) {
+
+		// added to recalculate if overlapping
+		this.update(labels);
```

Comment:
@simonbrunel this update has been added because when you have more than 1 labels and at the beginning of the drawing they are overlapped, the label is not longer shown, until next chart.update. I had an sample with bar chart with floating bars where it was happening. I'm gonna create again the use case because some time is gone and I have to recreate.

Reply:
It sounds like a bug not related to v3, right? If this issue was already in v2.9, let's fix it in a separate PR. If related to v3, then we will need to find another solution because calling `update()` in `draw()` is a no go.

Reply:
> 
> 
> It sounds like a bug not related to v3, right? If this issue was already in v2.9, let's fix it in a separate PR. If related to v3, then we will need to find another solution because calling `update()` in `draw()` is a no go.

No, in version 2.9.3 it works, because it works right now on my [showcase](http://www.pepstock.org/Charba-Showcase/Charba_Showcase.html). Let me take time to reproduce the use case,
Unfortunately some time is gone and now I'm working on other stuff.

Reply:
Indeed, a reproduction case would help to figure out if the issue is in this plugin or in the core library. If there is no obvious acceptable fix, I would suggest to remove that line and open a new issue so we can fix it later in a better way.

Reply:
fixed (I'm gonna push asap)

Reply:
Removed the line and added chart.render() as discussed in https://github.com/chartjs/chartjs-plugin-datalabels/pull/203#discussion_r523190841